contents/blog/the-posthog-array-1-36-0.mdx

```diff
@@ -0,0 +1,157 @@
+
+You'll notice an updated color palette in most Insights (specifically: Trends, Funnels, Retention, and Stickiness). It's more pleasing to the eye, but it's not just about aesthetics – accessibility is equally important, and the new scheme is simultaneously much more colorblind-friendly.
+
+<img src={DataColorsImage} alt="A funnel with new data colors" style={{ width: 640 }} />
+
+### Improved: Switch between series in the persons modal
+
+Thanks to the new series selector, the persons modal is easier to use with multi-series Trends insights. Switch between series without ever having to close the window.
+
+<img src={MultiPointPersonsModalImage} alt="Switching between series in the persons modal" style={{ width: 621 }} />
+
+### Improved: Preflight checks
+
+When self-hosting PostHog, it's crucial that all components are up and running, which is why the preflight checks screen is the first thing you see when setting up a fresh instance. Our new version of this screen makes initial setup of your instance all the more pleasant.
+
+<img src={PreflightImage} alt="New preflight checks screen" style={{ width: 627 }} />
+
+### New: Restack joins the PostHog Marketplace
+
+<iframe
+    width="560"
+    height="315"
+    src="https://www.youtube.com/embed/clVpovL0vxI"
+    title="YouTube video player"
+    frameborder="0"
+    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+    allowfullscreen
+></iframe>
+
+[Restack](https://posthog.com/marketplace/restack) is a platform for deployiong open-source software stacks on Kubernetes and is run by a team of open-source enthusiasts. It's a natural match for [the PostHog Marketplace](https://posthog.com/marketplace/), where we've steadily grown the number of businesses offering services and support over the last few months.
+
+Restack enables you to deploy and benefit from a self-hosted PostHog instance, but without the maintenance busy work. Best of all, Restack Console can ensure that deployments are [HIPAA](/docs/privacy/hipaa-compliance), FISMA and SOC2 compliant — handy if you deal with a lot of [PII](/blog/what-is-personal-data-pii). Find out more in [the Marketplace, or head to [Restack's store listing for PostHog](https://www.restack.io/store/posthog) to find more great videos like the one above!
+
+### Other improvements & fixes
+
+Version 1.36 also adds hundreds of other improvements and fixes, including...
+
+-   **New**: Median conversion time is now presented in funnel insights, offering results that are more resistant to being skewed by outliers. The average continues to be available in detailed results. [#9810](https://github.com/PostHog/posthog/pull/9810)
+-   **Improved**: The in-app plugin source code editor has been divided into tabs, to make editing `plugin.json` and `index.ts` slightly more intuitive. Additionally, code is now automatically formatted with `prettier` on save. [#9795](https://github.com/PostHog/posthog/pull/9795)
```

Comment:
CC @mariusandra If you want to add anything else.

Reply:
"more intuitive" is a nice reframe of "less frustrating". Nothing to add.